src/croniter/croniter.py

```diff
@@ -401,36 +403,32 @@ def _get_next(
                 dom_dow_exception_processed = True
 
         if not dom_dow_exception_processed:
-            result = self._calc(self.cur, expanded,
-                                nth_weekday_of_month, is_prev)
+            result = self._calc(self.cur, expanded, nth_weekday_of_month, is_prev)
 
         # DST Handling for cron job spanning across days
         dtstarttime = self._timestamp_to_datetime(self.dst_start_time)
-        dtstarttime_utcoffset = (
-            dtstarttime.utcoffset() or datetime.timedelta(0))
+        dtstarttime_utcoffset = dtstarttime.utcoffset() or datetime.timedelta(0)
         dtresult = self._timestamp_to_datetime(result)
         lag = lag_hours = 0
         # do we trigger DST on next crontab (handle backward changes)
         dtresult_utcoffset = dtstarttime_utcoffset
         if dtresult and self.tzinfo:
             dtresult_utcoffset = dtresult.utcoffset()
-            lag_hours = (
-                self._timedelta_to_seconds(dtresult - dtstarttime) / (60 * 60)
-            )
-            lag = self._timedelta_to_seconds(
-                dtresult_utcoffset - dtstarttime_utcoffset
-            )
+            lag_hours = self._timedelta_to_seconds(dtresult - dtstarttime) / (60 * 60)
+            lag = self._timedelta_to_seconds(dtresult_utcoffset - dtstarttime_utcoffset)
```

Comment:
Oops a few things snuck in that aren't related to `if` formatting, let me fix

Reply:
well, we can let them in, idea is to incremental merge to review not a big block at once.

Reply:
Fixed them already :)